test/unit/report_importer_test.rb

```diff
@@ -10,7 +10,7 @@ class ReportImporterTest < ActiveSupport::TestCase
   end
 
   test 'it should import reports with no metrics' do
-    r = ReportImporter.import(read_json_fixture('report-empty.json'))
+    r = ConfigReportImporter.import(read_json_fixture('report-empty.json'))
     assert r
     assert_equal({}, r.metrics)
```

Comment:
So, when changing it back to ReportImporter a `NotImplementedError` will be raised because on the Report model we return `NotImplementedError` on `report_name_class`.
We could use a temporary subclass, but then what is the difference in using `ConfigReportImporter` which is a subclass?